rules/brand-more-than-logo/rule.md

```diff
@@ -0,0 +1,74 @@
+<script async src="//www.instagram.com/embed.js"></script>
+**Video: Your brand is more than just a logo (40 sec)**
+
+## Key elements of brand identity
+
+### 1. Color Palette: The Emotional Trigger
+Your brand colors evoke specific emotions. Consistent use helps build recognition and trust. Examples include:
+- **Blue:** Trust and stability (e.g., IBM, Visa).
```

Comment:
Hey @bettybondoc 

- [ ] 1. Please remove full stops from lists https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists
- [ ] 2. Could we add emojis next to these items? 🟩 🟦 🟥